internal/provider/gateway_fqdn.go

```diff
@@ -76,7 +76,22 @@ func NewGatewayFQDNDeployer(ctx context.Context, d *schema.ResourceData, threefo
 	return deployer, nil
 }
 
-func (k *GatewayFQDNDeployer) Validate(ctx context.Context, sub subi.SubstrateExt) error {
+func (k *GatewayFQDNDeployer) Validate(ctx context.Context, sub subi.SubstrateExt, nodeID uint32) error {
```

Comment:
nodeID is already stored in the GatewayFQDNDeploy and there is no need to pass it, no??